### ISSUE TITLE
Character check events bind to the hidden `textarea` input when Rich Text is enabled on Multiline fieldtypes.

### ISSUE THREAD

**Issue by taylordaughtry** [verbb/formie#1237]
### Describe the bug

Hi! In Formie `1.6.21`, Multi-Line fieldtypes with Rich Text and 'Limit Field Content' enabled don't properly validate length. 
Based on some quick investigation, it seems that the event binding is happening before the rich text's JS has initialized, instead binding to the `textarea` element that's hidden when Tiptap initializes. 

At the risk of not having enough context in this area of the plugin—perhaps a quick fix could be to change when the `initTextMax()` method was fired [here](https://github.com/verbb/formie/blob/9d893498b29b081841e93a63a4aff624347c0059/src/web/assets/frontend/src/js/fields/text-limit.js#L18), ensuring Tiptap has already loaded?

### Steps to reproduce

1. Add a Multi-Line Text field with the following settings:
    - Limit Field Content enabled, any character limit value
    - Rich Text enabled
3. On the frontend, when editing content within the field you shouldn't see the character count being updated, beyond the initial count of the field's existing value, if any.

### Form settings

- Multi-page form: Yes
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: No

### Craft CMS version

3.7.54

### Plugin version

1.6.21

### Multi-site?

Yes

**Comment by engram-design** [comment]
So the reason this hasn't been implemented was that we can't really test the length of rich text content fields - at least not in the way that people would expect - because it would contain HTML markup in the length calculations. `<strong>Some text</strong>` will fall above a 10 character limit.

But there was an issue where the value wasn't updating at all, which was fixed in Formie 2 (backtracked to Formie 1 now).

Both should be fixed for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.6.21",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.6.22](https://github.com/verbb/formie/releases/tag/1.6.22)
